.github/workflows/nightly-normal-session-e2e-tests.yml

```diff
@@ -371,6 +371,31 @@ jobs:
           artifact-aleph-e2e-client-image: ${{ needs.build-aleph-e2e-client-image.outputs.artifact-name-image }}
           timeout-minutes: 35
 
+  run-e2e-all-validators-have-ideal-performance:
+    needs:
+      - build-synthetic-node
```

Comment:
Instead of this job, we should depend on `build-production-aleph-node`.

Reply:
done